### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by anotherkassio**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
 # smallrepo
+
+foo
```
Comment (kassio) [inline README.md:3]: diff note line 3

Comment (anotherkassio) [reply README.md:3]: reply to diff note on line 3

File: README.md [hunk 2]
```diff
@@ -1 +1,3 @@
 # smallrepo
```
Comment (anotherkassio) [inline README.md:1]: diff note line 1

Comment (kassio) [reply README.md:1]: reply diff note line 1

**Review by gitlabkassio — COMMENTED** [review]
REVIEW

File: README.md [hunk 3]
```diff
@@ -1 +1,3 @@
...
+
```
Comment (anotherkassio) [inline README.md:2]: diff note for review

**Review by anotherkassio — COMMENTED** [review]
review comment with diff note

**Comment by anotherkassio** [comment]
Issue comment?

**Pull request merged**